suite/performance/runlog.go

```diff
@@ -171,27 +174,74 @@ func (f *RunlogTest) Run() error {
 	if err := f.createChainlinkJobs(); err != nil {
 		return err
 	}
-	ctx, cancel := context.WithTimeout(context.Background(), f.TestOptions.TestDuration)
-	defer cancel()
-	i := 1
+	var ctx context.Context
+	var testCtxCancel context.CancelFunc
+	if f.TestOptions.TestDuration.Seconds() > 0 {
+		ctx, testCtxCancel = context.WithTimeout(context.Background(), f.TestOptions.TestDuration)
+	} else {
+		ctx, testCtxCancel = context.WithCancel(context.Background())
+	}
+	defer testCtxCancel()
+	cancelPerfEvents := f.watchPerfEvents()
+	currentRound := 0
 	for {
 		select {
 		case <-ctx.Done():
 			log.Warn().Msg("Test finished")
+			time.Sleep(f.TestOptions.GracefulStopDuration)
+			cancelPerfEvents()
 			return nil
 		default:
-			log.Warn().Int("RoundID", i).Msg("New round")
+			log.Warn().Int("RoundID", currentRound).Msg("New round")
```

Comment:
Why does this necessitate a `WARN` log?

Reply:
Just visually better, as a marker for long tests, can switch to INFO